.circleci/config.yml

```diff
@@ -34,18 +34,8 @@ jobs:
                                   gcab
                                   ca-certificates
                                   python3-pip
-                                  mingw32-nsis
-                                  mingw64-brotli
-                                  mingw64-gcc
-                                  mingw64-gettext
-                                  mingw64-pkg-config
-                                  mingw64-glib2
-                                  mingw64-gnutls
-                                  mingw64-libusbx
-                                  mingw64-sqlite
-                                  mingw64-libarchive
-                                  mingw64-json-glib
-                                  mingw64-curl
+                                  tree
```

Comment:
shouldn't everything else above here be in dependencies.xml?  The idea is supposed to be you don't have to manually install anything

Reply:
Yes, and done. There are a few more native things than I liked, but I've been filing bugs against other components all day. As they get merged and we update the subprojects we can remove a few BRs from the mingw64 build.